packages/commerce-sdk-react/src/hooks/query.test.tsx

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import nock from 'nock'
+import {renderHookWithProviders, DEFAULT_TEST_HOST} from '../test-utils'
+import {
+    useBasket,
+    usePaymentMethodsForBasket,
+    usePriceBooksForBasket,
+    useShippingMethodsForShipment,
+    useTaxesFromBasket
+} from './ShopperBaskets/query'
+import {useProductSearch, useSearchSuggestions} from './ShopperSearch/query'
+import {
+    useExternalProfile,
+    useCustomer,
+    useCustomerAddress,
+    useCustomerBaskets,
+    useCustomerOrders,
+    useCustomerPaymentInstrument,
+    useCustomerProductLists,
+    useCustomerProductList,
+    useCustomerProductListItem,
+    usePublicProductListsBySearchTerm,
+    usePublicProductList,
+    useProductListItem
+} from './ShopperCustomers/query'
+import {useOrder, usePaymentMethodsForOrder, useTaxesFromOrder} from './ShopperOrders/query'
+import {useProducts, useProduct, useCategories, useCategory} from './ShopperProducts/query'
+import {usePromotions, usePromotionsForCampaign} from './ShopperPromotions/query'
+
+jest.mock('../auth/index.ts', () => {
+    return jest.fn().mockImplementation(() => ({
+        ready: jest.fn().mockResolvedValue({access_token: '123'})
+    }))
+})
+
+const QUERY_TESTS = [
+    {
+        name: 'useBasket',
+        hook: () => useBasket({basketId: 'test'}),
+        endpoint: '/baskets/test'
+    },
+    {
+        name: 'usePaymentMethodsForBasket',
+        hook: () => usePaymentMethodsForBasket({basketId: 'test'}),
+        endpoint: '/baskets/test/payment-methods'
+    },
+    {
+        name: 'usePriceBooksForBasket',
+        hook: () => usePriceBooksForBasket({basketId: 'test'}),
+        endpoint: '/baskets/test/price-books'
+    },
+    {
+        name: 'useShippingMethodsForShipment',
+        hook: () => useShippingMethodsForShipment({basketId: 'test', shipmentId: '123'}),
+        endpoint: '/baskets/test/shipments/123/shipping-methods'
+    },
+
+    // TODO: not implemented
```

Comment:
i'm going to create another PR to consistently handle those "not implemented" hooks.

Reply:
What is the plan for documenting this? Can you explain it at a high level so we're all aligned. I had feedback on @yunakim714's PR and we should standardize on the approach for documenting the "not implemented" group